src/backend/upload_file/upload_file_for_chunks.py

```diff
@@ -0,0 +1,172 @@
+import json
```

Comment:
Add an issue in github isses and add attachments of the following test files you have done

Download four complex pdf files from google and extract those files raw_content

Upload the pdf files and the corresponding rawcontent file to the github issues that you have created and reply with the github issue. I will check those 4 complex file and whether we have extracted clearly